utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/encode/AvroSchemaOutputValidatorPrinter.scala

```diff
@@ -48,7 +49,7 @@ object AvroSchemaOutputValidatorPrinter {
       case Schema.Type.ARRAY =>
         s"List[${print(schema.getElementType)}]"
       case Schema.Type.MAP =>
-        s"Map[String, ${print(schema.getValueType)}]"
+        s"Map[String,${print(schema.getValueType)}]"
```

Comment:
Please also remove 'space' from record printing?

Reply:
In TypedObjectTypingResult we have version with space:
```
fields.map { case (name, typ) => s"$name: ${typ.display}"}.mkString("{", ", ", "}")
```.

So we are consistent here.